Bmx.Service.Aws/AwsStsClient.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SecurityToken;
+using Amazon.SecurityToken.Model;
+
+namespace Bmx.Service.Aws {
+	internal class AwsStsClient : IAwsStsClient {
```

Comment:
Is this just a wrapper around `AmazonSecurityTokenServiceClient`? Why not directly use [IAmazonSecurityTokenService](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/SecurityToken/TISecurityTokenService.html)?